sw/device/tock/dif/build.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
```

Comment:
Please add a comment explaining how this file works and what it expects as input. It's very clearly a file that assumes Meson is running.

Reply:
Apologies, missed this comment - in process of adding it.

Reply:
Done, also ran rustfmt.